client/data/launchpad-navigator/use-launchpad-navigator.ts

```diff
@@ -0,0 +1,44 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import wpcomRequest, { canAccessWpcomApis } from 'wpcom-proxy-request';
+
+export interface APIFetchOptions {
```

Comment:
Should we import this type from `@wordpress/api-fetch`? It doesn't feel right to (re)declare this locally.

Reply:
Last time I checked, the `@wordpress/api-fetch` package wasn't exporting its types. Could be a good opportunity to fix that in the meantime.

Reply:
You're right, @tyxla -- it looks like it _has_ types, but the main file is vanilla JS, so they're not a clear part of the package/exports. We'll have to tackle that as a separate/dedicated update upstream.